app/layout.tsx

```diff
@@ -20,12 +20,10 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en" className={GeistSans.className}>
-      <body className="text-foreground">
+      <body className="dark:dark bg-background px-4 pb-8 text-foreground">
```

Comment:
Q: dark: dark is just dark mode, correct?

Reply:
Correct. `dark: dark` applies the `dark` class established in `global.css`